bin/delete_data.py

```diff
@@ -64,13 +65,16 @@ def deletable_runs(self):
 
         deletable_runs = []
         for r in runs:
-            review_statuses = r.get('review_statuses')
-            most_recent_proc = r.get(ELEMENT_PROCS, [{}])[-1]
-            if type(review_statuses) is list:
-                review_statuses = [s for s in review_statuses if s]
-            if review_statuses and 'not reviewed' not in review_statuses:
-                if most_recent_proc.get(ELEMENT_STATUS) in ('finished', 'aborted'):  # i.e. not 'deleted'
-                    deletable_runs.append(r)
+            if self.list_runs and r[ELEMENT_RUN_NAME] in self.list_runs:
+                deletable_runs.append(r)
+            else:
+                review_statuses = r.get('review_statuses')
+                most_recent_proc = r.get(ELEMENT_PROCS, [{}])[-1]
+                if type(review_statuses) is list:
+                    review_statuses = [s for s in review_statuses if s]
+                if review_statuses and 'not reviewed' not in review_statuses:
+                    if most_recent_proc.get(ELEMENT_STATUS) in ('finished', 'aborted'):  # i.e. not 'deleted'
+                        deletable_runs.append(r)
 
```

Comment:
What do we want the behaviour of `--list_runs` to be? Should it force-delete runs, or select specific runs to delete?

At the moment, if you tell it to delete a specific run, it will pick up that, but also all other deletable runs anyway